requirements-dev.in

```diff
@@ -6,7 +6,8 @@ pytest-xdist
 
 freezegun
 responses
-factory_boy
+# factory_boy
+git+https://github.com/FactoryBoy/factory_boy.git#factory_boy
```

Comment:
why the git repository?

Reply:
There is crash when locale is not specified otherwise. It is fixed in master but not in the released version yet.